documentation/src/docs/pravega-concepts.md

```diff
@@ -11,429 +11,312 @@ You may obtain a copy of the License at
 
-The State Synchronizer can occasionally be "compacted", compressing and removing
-old state updates so that only the most recent version of the state is kept in
-the backing stream.  This feature helps app developers make sure that shared
-state does not grow unchecked.
-
- 
+The State Synchronizer can occasionally be "compacted", by compressing and removing
+older updates and retains only the recent version of the state in the backing Stream. This feature assures the application developers, that the shared state does not grow unchecked.
 
-State Synchronizer works best when most updates to shared state are small in
-comparison to the total data size being stored, allowing them to be written as
+State Synchronizer works effectively when most updates to shared state are small in
+comparison to the total data size being stored. This can be achieved by allowing them to be written as
 small deltas. As with any optimistic concurrency system, State Synchronizer is
-not at its best when many processes are all attempting to simultaneously update
-the same piece of data.
+not at its best when many processes attempts for simultaneous updates on the same piece of data.
 
-For more details on working with State Synchronizers, see [Working with Pravega:
+For more details on working with State Synchronizers, please see [Working with Pravega:
 State Synchronizer](state-synchronizer.md).
 
 ## Architecture
 
-The following figure depicts the components deployed by Pravega:
+The following figure depicts the components deployed by Pravega
 
 ![pravega high level architecture](img/pravega.arch.new.png)
 
 Pravega is deployed as a distributed system – a cluster of servers and storage
-coordinated to run Pravega called a "Pravega cluster".  
+coordinated to run Pravega called a **Pravega cluster**.  
 
-Pravega presents a software-defined storage (SDS) architecture formed by Controller instances 
-(control plane) and Pravega Servers (data plane).The set of Pravega Servers is known collectively as the Segment Store. 
+Pravega presents a software-defined storage (SDS) architecture formed by **Controller** instances
+(_control plane_) and Pravega Servers (_data plane_).The set of Pravega Servers is collectively known as the **Segment Store**. 
 
-The set of Controller instances make up the control plane of Pravega, providing
-functionality to create, update and delete Streams, retrieve information about
-Streams, monitor the health of the Pravega cluster, gather metrics etc.  There
-are usually multiple (recommended at least 3) Controller instances running in a
+The set of Controller instances together forms the control plane of Pravega, providing
+functionality to _create, update_ and _delete_ _Streams_. Further, it extends the functionality to retrieve information about the Streams, monitor the health of the Pravega cluster, gather metrics, etc.,. A minimum requirement of three or multiple Controller instances are allowed to be running in a
```

Comment:
I suggest to remove italics from _Streams_.

Reply:
`etc.,.` -> `etc.`

Reply:
`A minimum requirement of three or multiple` -> three are already multiple, so this sentence is confusing. If possible, revert to the previous one that seems clear enough to me.

Reply:
Accepted Suggestion